docs/reference/requests/README.md

```diff
@@ -27,120 +14,3 @@ event in the request.
-subsequent transfers after the chain. The event that was the first to fail within a chain will have
-a unique error result. Other events in the chain will have their error result set to
-[`linked_event_failed`](./create_transfers.md#linked_event_failed).
-
-### Linked Transfers Example
-
-Consider this set of Transfers as part of a request:
-
-| Transfer | Index in Request | flags.linked |
-| -------- | ---------------- | ------------ |
-| `A`      | `0`              | `false`      |
-| `B`      | `1`              | `true`       |
-| `C`      | `2`              | `true`       |
-| `D`      | `3`              | `false`      |
-| `E`      | `4`              | `false`      |
-
-If any of transfers `B`, `C`, or `D` fail (for example, due to
-[`exceeds_credits`](./create_transfers.md#exceeds_credits)), then `B`, `C`, and `D` will all fail.
-They are linked.
-
-Transfers `A` and `E` fail or succeed independently of `B`, `C`, `D`, and each other.
-
-After the chain of linked events has executed, the fact that they were linked will not be saved. To
-save the association between Transfers or Accounts, it must be
-[encoded into the data model](../../coding/data-modeling.md), for example by adding an ID to one of
-the [user data](../../coding/data-modeling.md#user_data) fields.
-
-## Guarantees
-
-- A request executes within the cluster at most once.
-- Requests do not [time out](../sessions.md#retries). Clients will continuously retry requests until
-  they receive a reply from the cluster. This is because in the case of a network partition, a lack
-  of response from the cluster could either indicate that the request was dropped before it was
-  processed or that the reply was dropped after the request was processed. Note that individual
-  [pending transfers](../../coding/two-phase-transfers.md) within a request may have
-  [timeouts](../transfer.md#timeout).
-- Requests retried by their original client session receive identical replies.
-- Requests retried by a different client (same request body, different session) may receive
-  different replies.
-- Events within a request are executed in sequence. The effects of a given event are observable when
-  the next event within that request is applied.
-- Events within a request do not interleave with events from other requests.
-- All events within a request batch are committed, or none are. Note that this does not mean that
-  all of the events in a batch will succeed, or that all will fail. Events succeed or fail
-  independently unless they are explicitly [linked](#linked-events)
-- Once committed, an event will always be committed — the cluster's state never backtracks.
-- Within a cluster, object
-  [timestamps are unique and strictly increasing](../../coding/time.md#timestamps-are-totally-ordered).
-  No two objects within the same cluster will have the same timestamp. Furthermore, the order of the
-  timestamps indicates the order in which the objects were committed.
```

Comment:
Did this section move somewhere else? I think it is still useful.

Reply:
Moved to coding!

https://github.com/tigerbeetle/tigerbeetle/blob/matklad/docs-reset/docs/coding/requests.md#guarantees

Reply:
Aha, thanks!